dialog/romimportsystem.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+_CONFIG=/opt/retronas/config/retronas.cfg
+source $_CONFIG
+source ${LIBDIR}/common.sh
+cd ${DIDIR}
+
+## If this is run as root, switch to our RetroNAS user
+DROP_ROOT
+CLEAR
+
+rn_import_system() {
+
+  local MENU_NAME=romimportsystem
+  READ_MENU_JSON "${MENU_NAME}"
+  READ_MENU_TDESC "${MENU_NAME}"
+  DLG_MENUJ "${MENU_TNAME}" 10 "${MENU_BLURB}"
+  
+
+    while read -r LINE; do
+        LINE_ARR=($LINE)
+        STR=${LINE_ARR[0]}
+        SUB="id"
+        if [[ "$STR" == *"$SUB"* ]]; then
+            SYSTEMS+=${LINE_ARR[1]}' '
+        fi
+    done < <(/opt/retronas/scripts/romimport.sh -l)
+
+    I=2
+    for S in $SYSTEMS:
+    do
+        SYSTEM_ARR[$I]=$S
+        ((++I))
+    done
+
+  while true
+  do
+    source $_CONFIG
+    
+    case ${CHOICE} in
+      01)
+        EXIT_OK
+      ;;
+      0[2-9]|[1-2][0-9]|30)
```

Comment:
I think this can be simplified to 

`[!01] | [0-9][0-9])`

Reply:
`[!01]`
Doesn't do what you think it does.
My understanding is that that syntax matches either ! or 0 or 1, as it is a character group construct borrowed from regular expressions.

Reply:
yes you are correct, it should be `! 01`

Reply:
arguably tho i think its first match anyway so we'd always match on 01 and break.

Reply:
I'm not sure `!01` will work either, but I think fallthrough is probably the best solution in this case anyways.